### ISSUE TITLE
SHARE image

### ISSUE THREAD

**Issue by szarnyasg** [martin-fleck/cra-ttc2016#5]
Hi,

I am working on a solution (with @n-a-sz) and would like to create a SHARE image. We were granted access to the TTC16 bundle, but the bundle seems empty. Are there available VMs yet? Or should we start off with an operation system such as Ubuntu 12.04 LTS?

Thanks,
Gabor

**Comment by szarnyasg** [comment]
In the meantime, a TTC16 Windows bundle named `TTC16::Win7_AD2SC` appeared. This seems like the image for the [PN2SC case](http://arxiv.org/pdf/1312.0342.pdf). Should we use this for the submission?

**Comment by pvgorp** [comment]
Hi all, please clone a VM from the Operating Systems bundle.  The base image from last year is also available there.

Regards,
Pieter

**Comment by fikovnik** [comment]
Hi,

here is a step-by-step guide that might help https://docs.google.com/document/d/1yN3l8k9oetWqo3c5RtwYCiW9yqbKn1C8upGkAD6Saac

Hope it will help

**Comment by pvgorp** [comment]
Hi Filip, I would suggest to add steps for joining also the Operating Systems bundle. Then, they can clone a VM from there.  Also, you may want to add links to the documentation pages for publishing and in particular the info on getting Bibtex for inclusion in the TTC papers.

Regards,
Pieter

**Comment by pvgorp** [comment]
Update: I have already updated the Google Doc with a few lines. In case anything is unclear then please let me know.  If some steps are trivial and messing up the clarity of the instructions then we would also like to hear that.

Regards

**Comment by szarnyasg** [comment]
Hey,

so to get this totally clear: we are free to choose our VM from the OS bundle: Arch, Win2003 or Ubuntu12?

G.

**Comment by pvgorp** [comment]
Indeed, and in case you wish to contribute new VDIs to the OS bundle we can even support that. The Arch VM was contributed by Tassilo Horn last year.  It has served many people very well but perhaps some of you want to invest some time in an even better base VM.

Kind regards

**Comment by pvgorp** [comment]
Please do note that we will move Windows based clones to a bundle which is only accessible to academic researchers.

**Comment by szarnyasg** [comment]
Just a heads up: according to our experience, the Ubuntu image uses quite a lot of RAM on its own (without a running Eclipse, etc.), so we'll go with the Arch image.

![collabshot_2016-4-3_1yhzl3](https://cloud.githubusercontent.com/assets/1402801/14987737/03ea9c94-1151-11e6-8d94-88764b8650f6.png)
